instructor-training-process.md

```diff
@@ -0,0 +1,154 @@
+	(because Data Carpentry doesn't teach Git,
+	and therefore doesn't want to require instructors to have to know it).
+    1.  Changes to content are discouraged,
+	since the lessons are now fairly stable.
+    1.  New exercises are also becoming problematic,
+	since we already have more than we can possibly use.
+    1.  Load on lesson maintainers has become unsustainable.
+1.  **Discussion sessions.**
+    1.  The trainee then takes part in an hour-long group discussion of that lesson
+        led by an experienced instructor.
+    1.  She is expected to have gone through the lesson *before* this session
+        so that she can ask lots of pointed questions during that hour.
+    1.  If the discussion leader feels the trainee is unprepared,
+        she may be asked to do some more work and try again.
+1.  **Final demonstration lessons.**
+    1.  The trainee then does a short demonstration lesson via screen sharing.
+    1.  If the examiner feels that she needs to do more work,
+        she will be given feedback and asked to try again.
+    1.  Trainees do not have to qualify separately for different topics within one Carpentry,
+        but *do* have to do a separate demonstration lesson for each Carpentry.
+1.  **Welcoming to the community.**
+    1.  Once the trainee has passed the demonstration lesson,
+        we generate a PDF certificate for the trainee by running a command-line script in
+        [the certification repository][cert-repo]
+	and send it to her along with instructions for the items below.
+    1.  The instructions sent to the new instructor tell her to:
+        1.  join the `discuss` and `instructors` lists
+	1.  fill in a form on AMY to tell us where she is, what she's comfortable teaching, etc., and
+	1.  send us a short biography and photo to add to [our website][team-page].
+    1.  We manually add the biography and photo to the website
+        (since they almost always require reformatting).
+
+## Resources
+
+1.  [AMY][amy] is the web application we use to track instructors and workshops.
+    Most of it is only accessible to administrators,
+    but it does have a form where instructors can supply or update information about themselves.
+1.  There's a web form on AMY that people can use to apply for instructor training
+    (currently disabled).
+1.  We keep a list of trainees in a Google Docs spreadsheet with the following fields,
+    which we upate manually:
+    1.  name
+    1.  contact email
+    1.  which training course they took part in
+    1.  the name of the Software Carpentry lesson they're concentrating on (if any)
+    1.  the name of the Data Carpentry lesson they're concentrating on (if any)
+    1.  the URL of their Software Carpentry exercise submission (if any)
+    1.  whether or not they have submitted something for Data Carpentry
+    1.  the date and moderator of the discussion session they have taken part in
+        *   we currently store both values in one column
```

Comment:
Absolutely - I'm hoping we drop the spreadsheet entirely when this moves into AMY.  Question is, what do we provide instead that doesn't require extra effort for bookkeeping?  As a discussion leader, what would you want to see on the web page so that you could mark people as ready to move on (or not)?